dashboard/src/main/auth/Login.tsx

```diff
@@ -146,7 +146,7 @@ export default class Login extends Component<PropsType, StateType> {
   };
 
   renderGoogleSection = () => {
-    if (this.state.hasGoogle) {
+    if (true || this.state.hasGoogle) {
```

Comment:
```suggestion
    if (this.state.hasGoogle) {
```
Everything looks awesome! Just one small change, can we remove this true?

Reply:
So sorry I missed this. It was just for testing locally. Thanks a lot for catching this